File: 575 - Distribute Candies.java
import java.util.HashSet;

class Solution {
    public int distributeCandies(int[] candyType) {
        HashSet<Integer> s = new HashSet<>();
        for (int i: candyType) {
            s.add(i);
        }
        int n = candyType.length / 2;
        int l = s.size();
        return l < n ? l : n;
    }
}

File: 575 - Distribute Candies.js
/**
 * @param {number[]} candyType
 * @return {number}
 */
var distributeCandies = function (candyType) {
    let s = new Set(candyType);
    let n = candyType.length / 2;
    let l = s.size;
    return l < n ? l : n;
};

a = 'nasdmals';
Number.parseInt